autoload/test/rust/cargonextest.vim

```diff
@@ -0,0 +1,127 @@
+if !exists('g:test#rust#cargonextest#file_pattern')
+  let g:test#rust#cargonextest#file_pattern = '\v\.rs$'
+endif
+
+if !exists('g:test#rust#cargonextest#test_patterns')
+  let g:test#rust#cargonextest#test_patterns = {
+        \ 'test': ['\v(#\[%(\w+::|rs)?test)'],
+        \ 'namespace': ['\vmod (tests?)']
+    \ }
+endif
+
+if !exists('g:test#rust#cargonextest#patterns')
+  let g:test#rust#cargonextest#patterns = {
+        \ 'test': ['\v\s*%(async )?fn\s+(\w+)'],
+        \ 'namespace': []
+    \ }
+endif
+
+function! test#rust#cargonextest#test_file(file) abort
+  if a:file =~# g:test#rust#cargonextest#file_pattern
+    if exists('g:test#rust#runner')
+        return g:test#rust#runner ==# 'cargonextest'
+    else
+      call system('cargo nextest')
+      return v:shell_error != 101
```

Comment:
This doesn't actually run tests, only shows an info screen like this
```
cargo-nextest 0.9.28
A next-generation test runner for Rust. <https://nexte.st>

USAGE:
    cargo nextest [OPTIONS] <SUBCOMMAND>

OPTIONS:
        --color <WHEN>            Produce color output: auto, always, never [env: CARGO_TERM_COLOR=]
                                  [default: auto]
    -h, --help                    Print help information
        --manifest-path <PATH>    Path to Cargo.toml
    -v, --verbose                 Verbose output [env: NEXTEST_VERBOSE=]
    -V, --version                 Print version information

CONFIG OPTIONS:
        --config-file <PATH>
            Config file [default: workspace-root/.config/nextest.toml]

        --tool-config-file <TOOL:ABS_PATH>
            Tool-specific config files

SUBCOMMANDS:
    archive    Build and archive tests
    help       Print this message or the help of the given subcommand(s)
    list       List tests in workspace
    run        Build and run tests
    self       Manage the nextest installation
```

There doesn't seem to be any other way to detect nextest than doing this, should I remove this and just check `g:test#rust#runner`?